src/pages/_projects-details.scss

```diff
@@ -2,151 +2,154 @@
 @use "../components/button";
 
-.projects-details .grid {
-  display: grid;
-  grid-template-columns: 350px 1fr 310px;
-}
-
-.projects-details .related-projects-list,
-.projects-details .related-projects-list .searchbar {
-  background-color: map-get(variables.$colors, "bg");
-  position: sticky;
-  align-self: start;
-}
-
-.projects-details .related-projects-list {
-  border-right: 1px solid map-get(variables.$colors, "border");
-  top: app-header.$app-header-height;
-}
-
-.projects-details .related-projects-list .searchbar {
-  top: 0;
-}
-
-.projects-details .related-projects-list .searchbar input {
-  padding-top: map-get(variables.$form-controls-padding-y, "lg");
-  padding-bottom: map-get(variables.$form-controls-padding-y, "lg");
-  border: none;
-}
-
-.projects-details .related-projects-list .searchbar,
-.projects-details .related-projects-list .related-projects-list-item {
-  border-bottom: 1px solid map-get(variables.$colors, "border");
-}
-
-.projects-details .related-projects-list {
-  overflow-y: auto;
-  height: calc(100vh - app-header.$app-header-height);
-}
-
-.projects-details .container {
-  padding: map-get(variables.$container-padding, "lg");
-  gap: 1rem;
-  overflow-x: hidden;
-}
-
-.projects-details .project .header,
-.projects-details .project,
-.projects-details .activity {
-  display: flex;
-}
-
-.projects-details .project,
-.projects-details .activity {
-  flex-direction: column;
-}
-
-.projects-details .project {
-  gap: 2.5rem;
-}
-
-.projects-details .project .header {
-  align-items: center;
-}
-
-.projects-details .project .mini-profile .title {
-  font-size: 1.25rem;
-}
-
-.projects-details .project .intro {
-  overflow-x: auto;
-}
-
-.projects-details .project .intro .lead,
-.projects-details .project .description .lead,
-.projects-details .project .checklist .lead {
-  margin-bottom: 0.8rem;
-}
+.projects-details {
+  .grid {
+    display: grid;
+    grid-template-columns: 350px 1fr 310px;
+  }
 
-.projects-details .stats {
-  gap: 5rem;
-}
+  .container {
+    padding: map-get(variables.$container-padding, "lg");
+    gap: 1rem;
+    overflow-x: hidden;
+  }
 
-.projects-details .project .checklist .progress-bar {
-  margin-bottom: 1rem;
-}
+  .related-projects-list {
+    background-color: map-get(variables.$colors, "bg");
+    position: sticky;
+    align-self: start;
+    border-right: 1px solid map-get(variables.$colors, "border");
+    top: app-header.$app-header-height;
+    overflow-y: auto;
+    height: calc(100vh - app-header.$app-header-height);
+
+    .searchbar {
+      background-color: map-get(variables.$colors, "bg");
+      border-bottom: 1px solid map-get(variables.$colors, "border");
+      position: sticky;
+      align-self: start;
+      top: 0;
+
+      input {
+        padding-top: map-get(variables.$form-controls-padding-y, "lg");
+        padding-bottom: map-get(variables.$form-controls-padding-y, "lg");
+        border: none;
+      }
+    }
+
+    .related-projects-list-item {
```

Comment:
selector contains duplication

Reply:
There is also   <code>.related-projects-list</code>   class,  and it may confuse you.  I couldn't find any other similar class.

Reply:
SELECTOR CONTAINS DUPLICATION